dref/scripts.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from pdf2image import convert_from_path
+
+
+def pdf_image_extractor(file):
+    pages = convert_from_path(file)
+    for i in range(len(pages)):
+        page = pages[i]
+        page.save(f'{file}_preview.png', 'PNG')
+        with open(f'{file}_preview.png') as f:
+            _, tail = os.path.split(f.name)
+            return os.path.join('dref/images', tail)
```

Comment:
Not sure how this works?

Reply:
we are generating image from a pdf file uploaded, the image is the first page of pdf

Reply:
I get that, but how are we storing the file? Using file storage? So, how the `page.save()` and `dref/images` works here? Will this work when we have azure/s3 storage?

Reply:
we are using file storage, `page.save()` will create a new image from pdf, we use that new file and store that in our database.

Reply:
Will this work when we have azure/s3 storage?

Reply:
yes, since this works like we upload file as in other file field